src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashBuilds.java

```diff
@@ -49,7 +49,9 @@ public void onCompleted(AbstractBuild build, TaskListener listener) {
             return;
         }
         Result result = build.getResult();
-        JenkinsLocationConfiguration globalConfig = new JenkinsLocationConfiguration();
+        // Note: current code should no longer use "new JenkinsLocationConfiguration()"
+        // as only one instance per runtime is really supported by the current core.
+        JenkinsLocationConfiguration globalConfig = JenkinsLocationConfiguration.get();
```

Comment:
We need to add a null check or upgrade to newer jenkins dependency.

My vote is for the former, since the later would reduce compatibility.